examples/views/dynamic_user_dropdown.py

```diff
@@ -0,0 +1,86 @@
+import discord
+
+bot = discord.Bot()
+
+# Defines a custom user specific dynamic dropdown
+# that the user can choose. The callback function
+# in the class is strictly to prevent the dreaded
+# interaction failed. The results are returned to
+# the calling function for further processing.
+
+
+def external_user_specific_list_function(discord_id):
+    # Replace this example with whatever your source is
+    # for the users dynamically generated items
+    users_menu_items = []
+    for i in range(5):
+        users_menu_items.append(
+            f"Item #{i + 1}) - Dynamically Generated for <@{discord_id}>"
+        )
+        i += 1
+    return users_menu_items
+
+
+class Dropdown(discord.ui.Select):
+    def __init__(self, interaction):
+        # In this example, we are passing in the interaction object from the calling interaction.
+        self.value = None
+        self.interaction = interaction
+        self.discord_user = interaction.user.id
```

Comment:
some of the attrs here aren't needed or used for anything
```suggestion
    def __init__(self, interaction):
        # In this example, we are passing in the interaction object from the calling interaction.
        self.discord_user = interaction.user.id
```